inference-engine/src/mkldnn_plugin/nodes/mkldnn_mvn_node.cpp

```diff
@@ -1429,4 +1475,47 @@ bool MKLDNNMVNNode::created() const {
     return getType() == MVN;
 }
 
+bool MKLDNNMVNNode::checkAxesSuitability(const std::shared_ptr<const ngraph::Node>& node) {
```

Comment:
It looks like this function could be rewritten with an algorithm that would check axesVal in more general fashion, m.b. some kind of a loop with counters. The point is that the routine in its current state requires two vector instances and two vector comparisons in a worst case, while it could be solved with one linear pass through the axesVal vector.

Reply:
Fixed